Sources/CoreMethods/CoreMethods.swift

```diff
@@ -191,6 +195,46 @@ public final class CoreMethods: Sendable {
             }
         )
     }
+
+    /// Gets available payment methods for a card BIN
+    ///
+    /// Retrieves a list of payment methods available for a specified card BIN
+    /// (first 6-8 digits of the card number).
+    ///
+    /// - Parameters:
+    ///   - bin: Bank Identification Number (first 6-8 digits of card number)
+    ///   - mode: The processing mode to use (default: .aggregator)
+    ///
+    /// - Returns: An array of ``PaymentMethod`` objects containing available payment methods
+    ///
+    /// - Throws:
+    ///   - .invalidURL: If the API endpoint URL is malformed
+    ///   - .networkError: If a connection to the API cannot be established
+    ///   - .decodingFailed(Error): If the response cannot be decoded
+    ///
+    public func paymentMethods(
```

Comment:
would it be good to have the names as actions like:
```suggestion
    public func getPaymentMethods(
```

Reply:
fix thanks